be/src/common/status.h

```diff
@@ -108,6 +108,10 @@ class Status {
     static Status DuplicateRpcInvocation(const Slice& msg) {
         return Status(TStatusCode::DUPLICATE_RPC_INVOCATION, msg);
     }
+    static Status JsonFormat(const Slice& msg) {
```

Comment:
suggest to rename JsonFormat to JsonFormatError, use a subject+predicate structure.

Reply:
done